featurevis/featurevis.py

```diff
@@ -1,10 +1,11 @@
 """
 The main file for the feature vis process
 """
+
 import tensorflow as tf
 from tensorflow import keras
 
-from luna.featurevis import images as imgs
+from featurevis import images as imgs
```

Comment:
Out of curiosity, why did you change that? I think the previous version was correct, no? At least if you use this as a library.

Reply:
I did this because I wrote my main.py file inside the library and this luna was redundant at that point.

Reply:
In that case, we should probably change it back.